cmd/openshift-install/create.go

```diff
@@ -116,14 +116,15 @@ var (
 					if err2 := logClusterOperatorConditions(ctx, config); err2 != nil {
 						logrus.Error("Attempted to gather ClusterOperator status after installation failure: ", err2)
 					}
-					bundlePath, err2 := runGatherBootstrapCmd(rootOpts.dir)
-					if err2 != nil {
-						logrus.Error("Attempted to gather debug logs after installation failure: ", err2)
-					}
 					logrus.Error("Bootstrap failed to complete: ", err.Unwrap())
 					logrus.Error(err.Error())
-					if err2 := service.AnalyzeGatherBundle(bundlePath); err2 != nil {
-						logrus.Error("Attempted to analyze the debug logs after installation failure: ", err2)
+					bundlePath, err2 := runGatherBootstrapCmd(rootOpts.dir)
+					if err2 == nil {
+						if err2 := service.AnalyzeGatherBundle(bundlePath); err2 != nil {
+							logrus.Error("Attempted to analyze the debug logs after installation failure: ", err2)
+						}
+					} else {
+						logrus.Error("Attempted to gather debug logs after installation failure: ", err2)
```

Comment:
nit: Let's use `err` within the if-block scopes.
```suggestion
					if bundlePath, err := runGatherBootstrapCmd(rootOpts.dir); err == nil {
						if err := service.AnalyzeGatherBundle(bundlePath); err != nil {
							logrus.Error("Attempted to analyze the debug logs after installation failure: ", err)
						}
					} else {
						logrus.Error("Attempted to gather debug logs after installation failure: ", err)
```

Reply:
Rerolled to use if-block `err`s with 38c1aec03 -> cc188c7fd.